fsspec_reference_maker/hdf.py

```diff
@@ -257,40 +269,10 @@ def storage_info(self, dset):
                 blob = dsid.get_chunk_info(index)
                 key = tuple(
                     [a // b for a, b in zip(blob.chunk_offset, chunk_size)])
-                stinfo[key] = {'offset': blob.byte_offset,
-                               'size': blob.size}
+                stinfo[key] = {'offset': blob.byte_offset, 'size': blob.size}
             return stinfo
 
 
-def chunks_info(zarray, chunks_loc):
```

Comment:
This is called in `tanslator()` under `if cinfo:`

Reply:
The `chunks_info()` function was duplicated in the code so I removed one copy.